src/main/java/org/example/algorithms/searching/LinearSearch_recursion.java

```diff
@@ -0,0 +1,40 @@
+package org.example.algorithms.searching;
+
+public class LinearSearch_recursion {
```

Comment:
Please rename _LinearSearch_recursion_ to _LinearSearchRecursion_ to use CamelCase ( not snake_case) for class name. Thanks.